src/TensorNetwork.jl

```diff
@@ -728,6 +728,7 @@ end
 
 function Serialization.serialize(s::AbstractSerializer, obj::TensorNetwork)
     Serialization.writetag(s.io, Serialization.OBJECT_TAG)
+    serialize(s, TensorNetwork)
```

Comment:
mmm I'm trying to understand this line. Is this needed in order to tell Julia that the object to deserialize is a `TensorNetwork` right?

Reply:
Yes exactly, this is what I have seen in similar posts (for [example](https://stackoverflow.com/questions/49007433/how-to-implement-custom-serialization-deserialization-for-a-struct-in-julia#:~:text=Here%20is%20the%20updated%20code%3A)).